<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWinTabBoxConfigForm</class>
 <widget class="QWidget" name="KWinTabBoxConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,2">
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Content</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0" rowspan="2" colspan="3">
         <widget class="QCheckBox" name="showDesktop">
          <property name="text">
           <string>Include &quot;Peek at Desktop&quot; entry</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="switchingModeCombo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Recently used</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stacking order</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QCheckBox" name="oneAppWindow">
          <property name="text">
           <string>Only one window per application</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QCheckBox" name="orderMinimized">
          <property name="text">
           <string>Order minimized windows after unminimized windows</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Sort order:</string>
          </property>
          <property name="buddy">
           <cstring>switchingModeCombo</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filter windows by</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="filterDesktops">
          <property name="text">
           <string>Virtual desktops</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="desktopFilter" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>24</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="currentDesktop">
             <property name="text">
              <string>Current desktop</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="otherDesktops">
             <property name="text">
              <string>All other desktops</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterActivities">
          <property name="text">
           <string>Activities</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="activityFilter" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>24</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="currentActivity">
             <property name="text">
              <string>Current activity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="otherActivities">
             <property name="text">
              <string>All other activities</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterScreens">
          <property name="text">
           <string>Screens</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="screenFilter" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>24</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="currentScreen">
             <property name="text">
              <string>Current screen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="otherScreens">
             <property name="text">
              <string>All other screens</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="filterMinimization">
          <property name="text">
           <string>Minimization</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="minimizationFilter" native="true">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>24</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="visibleWindows">
             <property name="text">
              <string>Visible windows</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="hiddenWindows">
             <property name="text">
              <string>Hidden windows</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Shortcuts</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Forward</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>All windows</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Reverse</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Forward</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Reverse</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="3">
         <widget class="QWidget" name="scCurrentReverseContainer">
          <layout class="QHBoxLayout" name="scCurrentReverseLayout">
           <item>
            <widget class="KKeySequenceWidget" name="scCurrentReverse" native="true"/>
           </item>
           <item>
           <widget class="QLabel" name="scCurrentReverseOr">
            <property name="text">
             <string>or</string>
            </property>
           </widget>
           </item>
           <item>
            <widget class="KKeySequenceWidget" name="scCurrentReverseAlternate" native="true"/>
           </item>
           <item>
            <spacer name="scCurrentReverseSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="1" colspan="3">
         <widget class="QWidget" name="scCurrentContainer">
          <layout class="QHBoxLayout" name="scCurrentLayout">
           <item>
            <widget class="KKeySequenceWidget" name="scCurrent" native="true"/>
           </item>
           <item>
           <widget class="QLabel" name="scCurrentOr">
            <property name="text">
             <string>or</string>
            </property>
           </widget>
           </item>
           <item>
            <widget class="KKeySequenceWidget" name="scCurrentAlternate" native="true"/>
           </item>
           <item>
            <spacer name="scCurrentSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Current application</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QWidget" name="scAllReverseContainer">
          <layout class="QHBoxLayout" name="scAllReverseLayout">
           <item>
            <widget class="KKeySequenceWidget" name="scAllReverse" native="true"/>
           </item>
           <item>
           <widget class="QLabel" name="scAllReverseOr">
            <property name="text">
             <string>or</string>
            </property>
           </widget>
           </item>
           <item>
            <widget class="KKeySequenceWidget" name="scAllReverseAlternate" native="true"/>
           </item>
           <item>
            <spacer name="scAllReverseSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QWidget" name="scAllContainer">
          <layout class="QHBoxLayout" name="scAllLayout">
           <item>
            <widget class="KKeySequenceWidget" name="scAll" native="true"/>
           </item>
           <item>
           <widget class="QLabel" name="scAllOr">
            <property name="text">
             <string>or</string>
            </property>
           </widget>
           </item>
           <item>
            <widget class="KKeySequenceWidget" name="scAllAlternate" native="true"/>
           </item>
           <item>
            <spacer name="scAllSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Visualization</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1">
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="effectCombo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>The effect to replace the list window when desktop effects are active.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="effectPreviewButton">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="icon">
              <iconset theme="view-preview"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="kcfg_ShowTabBox">
          <property name="text">
           <string/>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="kcfg_HighlightWindows">
          <property name="toolTip">
           <string>The currently selected window will be highlighted by fading out all other windows. This option requires desktop effects to be active.</string>
          </property>
          <property name="text">
           <string>Show selected window</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KKeySequenceWidget</class>
   <extends>QWidget</extends>
   <header>kkeysequencewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_HighlightWindows</tabstop>
  <tabstop>kcfg_ShowTabBox</tabstop>
  <tabstop>effectCombo</tabstop>
  <tabstop>effectPreviewButton</tabstop>
  <tabstop>switchingModeCombo</tabstop>
  <tabstop>showDesktop</tabstop>
  <tabstop>oneAppWindow</tabstop>
  <tabstop>filterDesktops</tabstop>
  <tabstop>currentDesktop</tabstop>
  <tabstop>otherDesktops</tabstop>
  <tabstop>filterActivities</tabstop>
  <tabstop>currentActivity</tabstop>
  <tabstop>otherActivities</tabstop>
  <tabstop>filterScreens</tabstop>
  <tabstop>currentScreen</tabstop>
  <tabstop>otherScreens</tabstop>
  <tabstop>filterMinimization</tabstop>
  <tabstop>visibleWindows</tabstop>
  <tabstop>hiddenWindows</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>filterDesktops</sender>
   <signal>toggled(bool)</signal>
   <receiver>desktopFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterActivities</sender>
   <signal>toggled(bool)</signal>
   <receiver>activityFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterScreens</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterMinimization</sender>
   <signal>toggled(bool)</signal>
   <receiver>minimizationFilter</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_ShowTabBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
